test/spec/features/copy-paste/BpmnCopyPasteSpec.js

```diff
@@ -71,6 +71,75 @@ describe('features/copy-paste', function() {
       }));
 
 
+      describe('should copy boundary events without host', function() {
+
+        it('should copy/paste', inject(function(elementRegistry, canvas, copyPaste) {
+
+          // given
+          var boundaryEvent = elementRegistry.get('BoundaryEvent_1'),
+              rootElement = canvas.getRootElement();
+
+          // when
+          copyPaste.copy(boundaryEvent);
+
+          var copiedElements = copyPaste.paste({
+            element: rootElement,
+            point: {
+              x: 1000,
+              y: 1000
+            }
+          });
+
+          // then
+          expect(rootElement.children).to.have.length(2);
+
+          expect(copiedElements).to.have.length(1);
+
+          expect(copiedElements[0].type).to.equal('bpmn:BoundaryEvent');
```

Comment:
This cannot be a boundary event, it must be an intermediate event.

Reply:
Fixed it now with @barmac comment, after the fix this test failed :)